### ISSUE TITLE
Pulsatile abdominal mass

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10762]
**Preferred term label:**
Pulsatile abdominal mass

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Abdominal aortic aneurysm

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001311
- label: Pulsatile abdominal mass
- parents: [Abdominal mass (HP:0031500)](https://hpo.jax.org/app/browse/term/HP:0031500)
- pmid: PMID:22644671
- def: An abnormal enlargement or swelling in the abdomen that pulses or throbs.
- comment: Clinical palpation of a pulsating abdominal mass alerts the clinician to the presence of a possible abdominal aortic aneurysm (AAA).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10762
- synonyms: Pulsating abdominal mass(NONE)

**Comment by pnrobinson** [comment]
Also:
Adding new term: 
- ID: HP:6001312
- label: Nonpulsatile abdominal mass
- parents: [Abdominal mass (HP:0031500)](https://hpo.jax.org/app/browse/term/HP:0031500)
- pmid: 
- def: An abnormal enlargement or swelling in the abdomen that does not pulse or throb.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10762
- synonyms:
